README.md

```diff
@@ -26,31 +26,40 @@
 
 # FingerprintJS Pro Chrome Extension Example
 
+Fingerprint is a device intelligence platform offering 99.5% accurate visitor identification.
+
 This repository contains an example Chrome extension that uses the FingerprintJS FingerprintJS Pro JavaScript agent.
+
+## Requirements
+
+- Chromium based browser (Chrome, Edge, Opera, etc.)
+
+## Installation
+
 You can install the example chrome extension from [Chrome Web Store](https://chrome.google.com/webstore/detail/fingerprintjs-example-bro/knppbjgkegnlbhddedbilnfmnkdocekn).
 
-### Why it exists
+## Why it exists
 
 The Chrome extension environment has limited capabilities for external scripts running within content scripts. This repository showcases different strategies on how FingeprintJS Pro can be used in this ecosystem. 
 
-### How it works
+## How it works
```

Comment:
It seems unnecessary to have a whole section for one sentence. Consider combining the two  Why and How sections into one "How it works" section

Reply:
Done!